draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -851,6 +853,16 @@ To prevent these types of linkability, various methods, including but not limite
 - Use advanced cryptographic schemes, outside the scope of this specification.
 - Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
 
+Colluding Verifier/Verifier pairs correspond to RP+RP'-U Unlinkability as defined in [@ISO.27551].
+
+## Issuer Issuing One Type of SD-JWT
+
+Issuer issuing only one type of SD-JWT might have privacy implications, because if the Holder has an SD-JWT issued by that Issuer, its type and claim names can be determined.
+
+For example, if the National Cancer Institute only issued SD-JWTs with cancer registry information, it is possible to deduce that the Holder owning its SD-JWT is a cancer patient.
+
+To mitigate this issue, a group of issuers may elect to use a common Issuer identifier, or use a group signature scheme instead of an individual signature.
```

Comment:
There's not any group signatures registered or spec'd for JWS/JWT (that I know of anyway) so that's not really a meaningful suggestion.

Reply:
agreed. though, we do say in section 9.4 `Use advanced cryptographic schemes, outside the scope of this specification`.

```suggestion
To mitigate this issue, a group of issuers may elect to use a common Issuer identifier. A group signature scheme outside the scope of this specification may also be used, instead of an individual signature.
```